Makefile

```diff
@@ -95,7 +95,7 @@ upgrade: ## update the pip requirements files to use the latest releases satisfy
 		echo ; \
 		echo "== $$f ===============================" ; \
 		echo "pip-compile -v --no-emit-trusted-host --no-index $$REBUILD --upgrade -o $$f.txt $$f.in"; \
-		pip-compile -v --no-emit-trusted-host --no-index $$REBUILD --upgrade -o $$f.txt $$f.in || exit 1; \
+		pip-compile -v -o $$f.txt $$f.in || exit 1; \
```

Comment:
1. An you explain why you don’t need what was here?
2. echo line needs to match. Could the command be in a variable rather than duplicated so that can’t happen?

Reply:
Good catch. This was incorrectly commited to this PR. This is a change that I make quite often in order to recompile requirements without upgrading them. Upgrading takes a very long time with my internet connection and there is no dedicated `make compile-requirements` command. I wonder how other developers do it?